native_client/dotnet/DeepSpeechClient/Models/Metadata.cs

```diff
@@ -6,9 +6,9 @@
     public class Metadata
     {
         /// <summary>
-        /// Approximated probability (confidence value) for this transcription.
+        /// Approximated confidence value for this transcription.
         /// </summary>
-        public double Probability { get; set; }
+        public double Confidence { get; set; }
```

Comment:
Need to change this to Confidence:
https://github.com/mozilla/DeepSpeech/blob/fcb9bf6d9fddd9ede5a5bba422d15277250218c4/native_client/dotnet/DeepSpeechConsole/Program.cs#L29